plugins/node/opentelemetry-instrumentation-hapi/package.json

```diff
@@ -42,7 +42,6 @@
   },
   "devDependencies": {
     "@hapi/hapi": "20.0.1",
-    "@hapi/podium": "4.1.1",
```

Comment:
I guess this needs to be kept here to avoid build problems caused by incompatible type definitions in various hapi components (see https://github.com/hapijs/hapi/issues/4240).

Reply:
Thanks for the pointer! I would've never figured that out.

Reply:
it would be nice if we could add comments in package.json